dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/CollectionRestRepository.java

```diff
@@ -41,6 +56,8 @@
 @Component(CollectionRest.CATEGORY + "." + CollectionRest.NAME)
 public class CollectionRestRepository extends DSpaceRestRepository<CollectionRest, UUID> {
 
+    private static final Logger log = Logger.getLogger(CollectionRestRepository.class);
```

Comment:
Sidenote (not a required change) for future development (please let your developers know).  Now that we use log4j v2, this statement can be simplified to:
`private static final Logger log = Logger.getLogger();`
I.e. you don't need to pass in the Class anymore.